### ISSUE TITLE
@Checkbox with @NotNull issue

### ISSUE THREAD

**Issue by PragneshHadiyel** [openanthem/nimbus-core#539]
# Issue Details

**Type of Issue**  (check one with "X")
```
[X] Bug Report => Please search GitHub for a similar issue or PR before submitting
[ ] Feature Request => Please ensure feature is not already in progress
[ ] Support Request => Please do not submit support requests here, instead see: https://discourse.oss.antheminc.com/
```

## Current Behavior

- If we have @Checkbox with @NotNull, the mandatory icon(asterisk) does not show up. 

## Expected Behavior

- If we have @Checkbox with @NotNull, mandatory icon has to appear next to the label configured.

### Code Snippet

# Environment Details
* **Nimbus Version:**
NIMBUS all versions

* **Browser:**
Google Chrome

**Comment by sandeepmantha** [comment]
Is the dataType of the parameter configured String or Boolean ? Can you paste the sample configuration that is used ?

**Comment by sandeepmantha** [comment]
The issue is that the mandatory asterisks icon does not show up when @NotNull is configured. This was an existing issue in the previous version but will be addressed soon.

**Comment by lopezton-anthem** [comment]
Fix checked into 1.3.0.M2.
